src/main/java/frc/trigon/robot/poseestimation/t265/T265.java

```diff
@@ -0,0 +1,65 @@
+package frc.trigon.robot.poseestimation.t265;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.networktables.*;
+import org.littletonrobotics.junction.Logger;
+
+public class T265 {
+    private static T265 INSTANCE;
```

Comment:
Why use a singleton? We don't use singletons anywhere else. Stay consistent.

Reply:
Because I don't think it should be a part of RobotContainer

Reply:
Same as the camera's aren't part of robotcontainer, it's not a main component and it's only used for the pose estimator